src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -740,6 +740,41 @@ void reorder_inputs::run(program& p, reorder_factory& rf) {
         }
     };
 
+    const auto reorder_eltwise = [&p](typed_program_node<eltwise>& eltwise_node) {
```

Comment:
`reorder_eltwise` is too general. maybe `reorder_eltwise_input_for_np_broadcast`